src/java/boa/compiler/SymbolTable.java

```diff
@@ -379,7 +380,9 @@ public static void resetTypeMap() {
 		types.put("float", new BoaFloat());
 		types.put("time", new BoaTime());
 		types.put("string", new BoaString());
-
+		types.put("LinearRegression", new BoaLinearRegression());
+		types.put("Model", new BoaModel());
```

Comment:
`Model` isnt a type in the language, is it?

Reply:
Yes, "Model" is a type.